index.js

```diff
@@ -213,6 +215,16 @@ Multifeed.prototype.replicate = function (opts) {
       var keys = values(self._feeds).map(function (feed) { return feed.key.toString('hex') })
       mux.haveFeeds(keys)
     })
+
+    if(opts.live) {
+      // Push session to _streams array
+      self._streams.push(mux)
+
+      // Register removal
+      mux.stream.on('end', function(err) {
```

Comment:
What about `on('error')`?

Reply:
Hmm.. must be my memory failing, but wasn't there a single event that got emitted regardless of success/error result? If not then I'll add on('error') as well.

Reply:
added the mux.stream.on('error' ...) handler